controllers/images/resolver_test.go

```
@@ -0,0 +1,308 @@
+		},
+		Data: map[string]string{
+			TrustyAIServiceImageKey: testImageFromCM,
+		},
+	}
+
+	scheme := runtime.NewScheme()
+	_ = corev1.AddToScheme(scheme)
+	fakeClient := fake.NewClientBuilder().WithScheme(scheme).WithObjects(configMap).Build()
+
+	// Set the environment variable
+	os.Setenv(RelatedImageTrustyAIService, testImageFromEnv)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)